File: Clase 5 - Estructuras Condicionales/if anidado.py
edad = 19
tieneLicencia = True

if edad >= 18:
  print("Es mayor de edad")
  if tieneLicencia:
    print("Tiene licencia, puede conducir")
  else:
    print("No tiene licencia, no puede conducir")
else:
  print("No es mayor de edad, no puede conducir")

File: Clase 5 - Estructuras Condicionales/if anidado.js
const edad = 15
const tieneLicencia = false

if (edad >= 18) {
  console.log('Es mayor de edad')
  if (tieneLicencia) {
    console.log('Tiene licencia, puede conducir')
  } else {
    console.log('No tiene licencia, no puede conducir')
  }
} else {
  console.log('No es mayor de edad, no puede conducir')
}
